colorcoder/utils/NumberValidator.java

```diff
@@ -0,0 +1,17 @@
+package colorcoder.utils;
+
+/**
+ * @author {@literal Jayaram Naveenkumar (jayaram.naveenkumar@in.bosch.com)}
+ */
+public final class NumberValidator {
```

Comment:
Number is a ploymorphic in nature and isValid Validates integer only !

Reply:
I got ur point, I will rename isValid to isValidColorCode and class name to Vliadator